AmericanMedicalAssociation/ama-style-guide-2
# ADF-141: Add Discard Changes modal

[User story ADF-141: Prototype "discard changes" modal](https://palantir.atlassian.net/browse/ADF-141)

## Description
* Create a "Discard changes modal"

## Test
* Visit `http://localhost:3000/?p=molecules-dr-finder-modal` and verify modal
* Visit `http://localhost:3000/?p=pages-dr-finder-with-modal` and verify modal
